steep_expectations.yml

```diff
@@ -1 +1,13 @@
---- []
+---
+- file: lib/ulid.rb
+  diagnostics:
+  - range:
+      start:
+        line: 311
+        character: 12
+      end:
+        line: 311
+        character: 16
+    severity: ERROR
+    message: Type `singleton(::Warning)` does not have method `warn`
+    code: Ruby::NoMethod
```

Comment:
I hope https://github.com/ruby/rbs/pull/1056 removes this 🙏

Reply:
https://github.com/kachick/ruby-ulid/commit/c5a9740324601e30545555d9be8001b0325e565f